src/components/ItemFilter/ItemFilter.css

```diff
@@ -1,23 +1,7 @@
 .filter {
-  position: relative;
   margin: 20px 0;
 }
 
-.filter__text-field {
-  padding-right: 3rem;
-}
-
-.filter__clear {
-  position: absolute;
-  bottom: var(--border-width);
-  right: var(--border-width);
-  width: 2.75rem;
-  height: 2.75rem;
-  font-size: inherit;
-  color: var(--charcoal);
-  border: none;
-}
-
```

Comment:
Why were these filter styles removed or where did they get moved to?

Reply:
@connietran-dev Because I used a "button in form field" style thing again for the copy button, I decided to put those styles in a reusable block instead of specific to the filter, that we I could use the same block for both and if we change it only change it once. The "button in form field" combo block is called `.form-group` with `.form-group__button` and `.form-group__field`. I put a comment above that section explaining the purpose of `.form-group, though open to suggestions on better naming!